opentracing/utils.go

```diff
@@ -0,0 +1,27 @@
+package opentracing
+
+import (
+	"encoding/base64"
+	"fmt"
+	"net/http"
+)
+
+const (
+	OpenTracingContextIDHeader = "OpenTracing-Context-Id"
```

Comment:
That's a good question. The API as its presently written does not actually have the `Recorder` doing the work of storing/fetching to/from HTTP headers (which is just one of many possible examples). So even though I agree the value for the key is not going to mean anything if the two communicating processes have different recorder types, I do think there's some marginal value in having a utils file (or even package) that does the grunt work of getting things on and off the wire for various protocols.

Or that was my thought. I.e., note how this file is part of the main `opentracing` package rather than the `dapperish` implementation example.